analytics/api/src/main/java/mil/nga/giat/geowave/analytic/partitioner/AdapterBasedPartitioner.java

```diff
@@ -108,7 +108,9 @@ protected NumericDataHolder getNumericData(
 				entry.data,
 				getIndex().getIndexModel());
 		final double[] thetas = getDistancePerDimension();
-		final MultiDimensionalNumericData primaryData = encoding.getNumericData(getIndex().getIndexModel().getDimensions());
+		final MultiDimensionalNumericData primaryData = encoding.getNumericData(getIndex()
+				.getIndexModel()
```

Comment:
This is a formatting only PR and I verified against master that there is no code change here (this line was already present), not sure why exactly it's showing up in the diff like this.